Travel Scheduler/Views/DetailHeaderCell.m

```diff
@@ -117,6 +117,8 @@ - (instancetype)initWithWidth:(int)width andPlace:(Place *)givenPlace
 - (void)customLayouts
 {
     self.image = makeSquareImage(self.width);
+    self.image.backgroundColor = [UIColor whiteColor];
+    self.image.image = [UIImage imageNamed:@"output-onlinepngtools.png"];
```

Comment:
Could consider changing the imageview property name such that reader isn't confused.